mitodl/ocw-hugo-projects
# set code syntax highlighting to use the 'colorful' colorscheme

#### Pre-Flight checklist


#### What are the relevant tickets?

see https://github.com/mitodl/ocw-hugo-themes/issues/500

#### What's this PR do?

This changes the setting for syntax highlighting to a scheme which works well w/ a white background.

#### How should this be manually tested?

Try rendering some markdown which has a code block w/ language set on it (e.g. using the ` ```python ` syntax). It should be highlighted in a friendly, readable colorscheme.
